views/courses/course.hbs

```diff
@@ -123,6 +123,12 @@
       {{/content}}
       {{/embed}}
 
+      {{#embed "lib/components/modal-form" class="import-modal-from-template"}}
+	  {{#content "fields"}}
+		{{> "courses/forms/form-import-topic-from-template"}}
+	  {{/content}}
+	  {{/embed}}
```

Comment:
looks like mixed indentation. Please use tabs here.

Reply:
surrounding code is also indented this way. one option could be to indent all code right.

Reply:
![image](https://user-images.githubusercontent.com/22987140/69056794-f5558b00-0a10-11ea-99b9-f8eb43233374.png)

Reply:
ok